pom.xml

```diff
@@ -31,7 +31,7 @@
   </scm>
 
   <properties>
-    <jenkins.version>1.580.3</jenkins.version>
+    <jenkins.version>1.590</jenkins.version>
```

Comment:
Just go to the next LTS line, 1.609.3 (or later, if important downstream projects are already on something newer anyway).